packages/extra/seaweedfs/values.yaml

```
@@ -31,3 +31,24 @@ storageClass: ""
 ##     replicas: 2
 ##     size: 10Gi
 zones: {}
+
+## @param externalClusters A map of external clusters for managing bucketclaims.
+## Example:
+## externalClusters:
+##   cluster1: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)